pkg/util/testutil/assert/assert.go

```diff
@@ -0,0 +1,25 @@
+// Package assert provides utilities for making assertions during tests.
```

Comment:
Is there any widely accepted testing repository that has assert equal/not equals we can use instead?

Reply:
https://github.com/stretchr/testify is the most common one as far as I know. I think some part of Go community discourages [assert libraries](https://go.dev/wiki/TestComments#assert-libraries). I'm not against it – I just didn't wanted to pull a big package for simple assertions. We also depend on https://github.com/onsi/gomega in our integration tests as it's used in Kubernetes testing framework. 

I think I can create an issue to standardizing all assertions to one package in the codebase.